.github/workflows/env-setup-delete-redis.yml

```diff
@@ -0,0 +1,34 @@
+name: Env Setup - Delete Redis in Openshift
+run-name: Env Setup - Delete Redis in Openshift from ${{ inputs.environment }} using ${{ inputs.gitRef }}
+
```

Comment:
Something that we missed in the existing actions, can we have the below `concurrency` added to all these related yml files to prevent them from being executed at the same time?

```
run-name: Env Setup - Delete Redis in Openshift from ${{ inputs.environment }} using ${{ inputs.gitRef }}

concurrency: redis-setup

on:
```